api/src/main/java/org/openmrs/attribute/Attribute.java

```diff
@@ -42,7 +42,7 @@
 	void setOwner(OT owner);
 	
 	/**
-	 * @return the AttributeType that controls this attribute's behavior
```

Comment:
@dkayiwa, Lol it's not a typo, is a slight grammatical adjustment. I made the noun come before the subject since it is referred in the third person :smile: